src/index.html

```diff
@@ -1,13 +1,24 @@
 <html>
 <head>
     <meta charset="utf-8">
+    <meta name="viewport" content="initial-scale=1" />
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```

Comment:
I use md-data-table (https://github.com/iamisti/mdDataTable) for displaying tables in angualr where jquery is required as a mandatory dependancy. Maybe you have some plugin recommandations for displaying tables in angualrJS ?